meica.py

```diff
@@ -154,6 +154,13 @@ def getdsname(e_ii,prefixonly=False):
 	if prefixonly: return dsprefix(dsname)
 	else: return dsname
 
+def isnumber(val,ret=False):
+	retval=None
+	try: retval=float(val)
+	except: return False
```

Comment:
Can you include the error you expect in your `except` block? I imagine it should be a `TypeError`, if a None value is passed.